### ISSUE TITLE
5D Mk3 - Camera Error

### ISSUE THREAD

**Issue by divijb** [v002/v002-Camera-Live#29]
I had this setup working not too long ago. I believe i upgraded my firmware recently, and now it is no longer working. I also tried updating to the newest version of Camera Live 8. Basically, after I start up Camera Live, I see it detect "Canon EOS 5D Mark III". I click on it, and in the tan status window it displays "Camera Error". When I open OBS, and add device as "Game Capture (Syphon)", I can see 5D Mark III as an option, but there is no vide. I have tried both Video and still live view mode, but there is no video that comes through

**Comment by divijb** [comment]
Update: for anyone who came here from:
https://reefbum.com/photography-videography/how-to-use-a-dslr-for-facebook-live/

I have been tinkering with this, on and off, which includes following the instructions from the above URL, for about a week now. After so much struggle, I accidently started up Camera live app without first starting up the EOS utility. This automatically opened the shutter for me and Camera live does indeed work and I also see a video in OBS.

TLDR: Do not use EOS Utility any more. Just start Camera live first
